docs/core/entity/switch.md

```diff
@@ -16,6 +16,13 @@ Properties should always only return information from memory and not do I/O (lik
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
 | is_on | boolean | **Required** | If the switch is currently on or off.
+
+## Deprecated Properties
+
+The following properties are deprecated and should not be used by new integrations it is recommended to migrate them to sensors, the deprecated properties will remain until at least the end of 2021.
```

Comment:
No need to include a timeline. Nothing breaks for integrations if we don't fetch their properties (different story for users).
```suggestion
The following properties are deprecated and should not be used by new integrations. Provide them as sensors instead.
```